dnf-behave-tests/dnf/comps-environment-list.feature

```diff
@@ -1,11 +1,15 @@
-@dnf5daemon
 Feature: Tests for environment list and info commands
 
 
 Background: Enable repo and mark an environment as installed
  Given I use repository "comps-group-list"
-   And I successfully execute dnf with args "group install test-environment"
+   And I create file "/usr/lib/sysimage/libdnf5/environments.toml" with
```

Comment:
Can you please add a comment here explaining that the config change is needed instead of calling `group install` because of the daemon? The same for the environments.

Reply:
Done.